git-ftp

```diff
@@ -1280,6 +1281,16 @@ set_current_branch() {
 	CURRENT_BRANCH="$current"
 }
 
+set_stash() {
+	local stash_config="$(get_config auto-stash)"
+	[ -n "$stash_config" ] && AUTO_STASH="$(boolean $stash_config)"
+	[ $AUTO_STASH -eq 1 ] && git stash push --include-untracked -q -m "Stashed by Git-ftp"
```

Comment:
The `push` option is new. My Git version doesn't know it yet. Maybe we should use `git stash save` for compatibility. Newer versions still recognise it.

Reply:
Maybe you should consider to update your git version? 😜😉

No, seriously. You're right, they still recognise it, but [`save` is deprecated](https://stackoverflow.com/a/47231547) since almost two years. So I think it’s better to use the future-proof command, isn’t it?

What we could do is check the git-version and say “Oh sorry, this option is not supported for your git version. Please update git or stash your changes manually.”

Reply:
There is a lot of value in long term support. Don't change a running system. That's why I'm working with Debian stable. No surprises, it just works. My current version will still be supported for another three years.

Uh, I just saw that a new Debian stable was released ten days ago. So yes, I will update my Git version along with everything else soon! :-)